core/src/main/scala/cats/Alternative.scala

```diff
@@ -24,35 +25,35 @@ import simulacrum.typeclass
     }
 
   /**
-   * Separate the inner foldable values into the "lefts" and "rights"
-   *
-   * Example:
-   * {{{
-   * scala> import cats.implicits._
-   * scala> val l: List[Either[String, Int]] = List(Right(1), Left("error"))
-   * scala> Alternative[List].separate(l)
-   * res0: (List[String], List[Int]) = (List(error),List(1))
-   * }}}
-   */
+    * Separate the inner foldable values into the "lefts" and "rights"
+    *
+    * Example:
+    * {{{
+    * scala> import cats.implicits._
+    * scala> val l: List[Either[String, Int]] = List(Right(1), Left("error"))
+    * scala> Alternative[List].separate(l)
+    * res0: (List[String], List[Int]) = (List(error),List(1))
+    * }}}
+    */
   def separate[G[_, _], A, B](fgab: F[G[A, B]])(implicit FM: Monad[F], G: Bifoldable[G]): (F[A], F[B]) = {
     val as = FM.flatMap(fgab)(gab => G.bifoldMap(gab)(pure, _ => empty[A])(algebra[A]))
     val bs = FM.flatMap(fgab)(gab => G.bifoldMap(gab)(_ => empty[B], pure)(algebra[B]))
     (as, bs)
   }
 
   /**
-   * Return ().pure[F] if `condition` is true, `empty` otherwise
-   *
-   * Example:
-   * {{{
-   * scala> import cats.implicits._
-   * scala> def even(i: Int): Option[String] = Alternative[Option].guard(i % 2 == 0).as("even")
-   * scala> even(2)
-   * res0: Option[String] = Some(even)
-   * scala> even(3)
-   * res1: Option[String] = None
-   * }}}
-   */
+    * Return ().pure[F] if `condition` is true, `empty` otherwise
```

Comment:
While we probably haven't been entirely consistent, I think that we've been using javadoc-style comments, so I think that `docstrings = JavaDoc` might be a good setting for us. Unless I'm wrong and that actually generates a bigger diff...

Reply:
👍

Reply:
The diff is much smaller now :)